app/models/metadata_field.rb

```diff
@@ -43,85 +125,10 @@ class MetadataField < ApplicationRecord
-  # RDS flavor/version doesn't support json types so this is string.
-  # t.string :options
-
-  # If true then only allow users to use the options (i.e. no freetext).
-  # t.integer :force_options
-
-  # A hash that maps host genome ids to an array of examples for that metadata field and host genome.
-  # Stored as JSON.
-  # If the key is "all", the examples apply to all host genomes.
-  # We separate by host genome because, for example, valid sample_types are different for human vs mosquito,
-  # and the examples should reflect this.
-  # t.string :examples
-
-  # +----------------------+
-  # |      All fields      |
-  # | +------------------+ |
-  # | |      Core        | |
-  # | | +--------------+ | |
-  # | | |   Default    | | |
-  # | | | +----------+ | | |
-  # | | | | Required | | | |
-  # | | | +__________+ | | |
-  # | | +______________+ | |
-  # | +__________________+ |
-  # +______________________+
-
-  # The core fields are basically the set of fields that we think are important/interesting/have
-  # curated ourselves. All user-created custom types will not be core (unless we make them).
-  # Ex: lat_lon could be a core field but not on new projects by default.
-  # t.integer :is_core
-
-  # Default fields are a subset of the core fields that will appear on a project when someone
-  # creates a project. These can be removed from a project.
-  # See add_default_metadata_fields in project.rb.
-  # t.integer :is_default
-
-  # Required fields are a subset of the core/default fields that cannot be removed from the
-  # project. This is the strictest level. Ex: collection_date, location, nucleotide_type, etc.
-  # t.integer :is_required
-
-  # Name of a group of fields for the frontend. Ex: Sample, Donor, Infection, Sequencing, etc.
-  # t.string :group
-
-  # Certain meta-fields are appropriate for different (and potentially multiple) hosts. E.g.
-  # "Discharge Date" is only for humans. Therefore host genomes have many meta-fields and
-  # meta-fields have many host genomes. This is a way of handling many-to-many /
-  # has_and_belongs_to_many in Rails.
-  # create_join_table :host_genomes, :metadata_fields
-
-  # User-defined meta-fields will belong to each (and potentially multiple) projects. So
-  # meta-fields have many projects and projects have many meta-fields.
-  # When a user creates a new project, they'll basically get a list of all the meta-fields marked
-  # "default". Then they can add and subtract from their set of meta-fields from there.
-  # create_join_table :projects, :metadata_fields
-
-  # Whether this metadata field should be added automatically to new host genomes.
-  # See add_default_metadata_fields in host_genome.rb.
-  # t.integer :default_for_new_host_genome, limit: 1, default: 0
+  def default_for_host_genome?
+    # for migration compatibility
+    respond_to?(:default_for_new_host_genome) && default_for_new_host_genome == 1
+  end
 
   def metadata_field_subsets
```

Comment:
Nit: Could we name this `metadata_field_validations`? I believe the Rails convention is to describe the property you are validating, but it's a little hard to put into words. (is_default is a "metadata-field field", so it would be `metadata_field_field_dependencies`). `subsets` feels confusing to me though and causes me to think the function is about something else.